### PR TITLE
US-1229 Fix do not override built history

### PR DESCRIPTION
**Pull request by laurgarn**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/dto/GroupAndReportInfos.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.gridsuite.modification.server.dto;
```
Comment (niconoir) [inline src/main/java/org/gridsuite/modification/server/dto/GroupAndReportInfos.java:1]: Missing license header

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java [hunk 2]
```diff
@@ -99,6 +99,8 @@ public class NetworkModificationService {
     private static final String NETWORK_MODIFICATION_REPORT_KEY = "NetworkModification";
     private static final String NETWORK_MODIFICATION_REPORT_NAME = "NetworkModification";
 
+    private static final Logger LOGGER = LoggerFactory.getLogger("details");
```
Comment (niconoir) [inline src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:102]: Was LOGGER created for debug purpose ?

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java [hunk 3]
```diff
@@ -941,6 +943,7 @@ public Flux<EquipmentDeletionInfos> deleteEquipment(UUID networkUuid, String var
     }
 
     private void sendReport(UUID reportUuid, ReporterModel reporter) {
+        LOGGER.info("About to add report of uuid {}", reportUuid);
```
Comment (niconoir) [inline src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:946]: Was this logged for debug purpose ?

File: src/main/java/org/gridsuite/modification/server/dto/BuildInfos.java [hunk 4]
```diff
@@ -31,12 +32,14 @@ public class BuildInfos {
...
     private String destinationVariantId;
 
-    private List<UUID> modificationGroupUuids = new ArrayList<>();
-
-    private List<UUID> reportUuids = new ArrayList<>();
+    private List<GroupAndReportInfos> modificationGroupAndReportUuids = new ArrayList<>();
 
     private Set<UUID> modificationsToExclude = new HashSet<>();
 
+    public List<UUID> getModificationGroupUuids() {
```
Comment (niconoir) [inline src/main/java/org/gridsuite/modification/server/dto/BuildInfos.java:39]: This method seems to be unused.

**Review by niconoir — CHANGES_REQUESTED** [review]
Minor comments.

File: src/main/java/org/gridsuite/modification/server/dto/GroupAndReportInfos.java (hunk 1 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/modification/server/dto/GroupAndReportInfos.java:7]: Added

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java (hunk 2 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: Removed

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java (hunk 3 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: Yes, removed

**Review by niconoir — APPROVED** [review]

**Pull request merged**